supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -509,6 +509,19 @@ There is no functional difference between the first server that was installed an
 
 *See also*:
 
+[discrete]
+[[intel-virtualization-technology]]
+==== image:images/yes.png[yes] Intel 64 (noun)
+*Description*: Intel's 64-bit version of the x86 architecture. Use when referring to information that is exclusive to Intel processors. Only use in RHEL documentation. 
+
+Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
+
```

Comment:
I found the examples in the other entries very helpful for understanding the context. Can you add one here as well?